database/query/datasources.sql

```diff
@@ -84,7 +84,7 @@ WHERE data_source_id = $1 AND project_id = $2;
 --
 -- name: ListRuleTypesReferencesByDataSource :many
 SELECT * FROM rule_type_data_sources
-WHERE data_sources_id = $1;
+WHERE data_sources_id = $1 and project_id = $2;
```

Comment:
`project_id` is pretty ambiguous... which is it? the project from the data source or the project from the rule type?